packages/instui-cli/lib/commands/create.js

```diff
@@ -47,22 +47,6 @@ exports.builder = (yargs) => {
     })
   }
 
-  // TODO: remove create app script and any related code
-  const appCommand = 'app'
```

Comment:
Another part to delete: 

In the README file of the package (`packages/instui-cli/README.md`), we use the app generator script as an example. Remove it from the "Available commands" block and replace the examples with another one, e.g. with the create component script: 

`npx @instructure/instui-cli create component`
`instui create component`